plugins/GSdx/res/tfx.fx

```diff
@@ -381,7 +381,10 @@ float4x4 sample_4p(float4 u)
 
 float4 sample(float2 st, float q)
 {
-	if(!PS_FST) st /= q;
+	if(PS_FST == 0)
```

Comment:
Expression before and after are the same.

Reply:
I wanted it to match gl code, guess it's a bit of an useless change.

Reply:
AFAICT this PR doesn't actually change anything.